frontend/src/component/feature/Dependencies/AddDependencyDialogue.tsx

```diff
@@ -197,21 +230,58 @@ export const AddDependencyDialogue = ({
                 <DependenciesUpgradeAlert />
                 <Box sx={{ mt: 2, mb: 4 }}>
                     Your feature will be evaluated only when the selected parent
-                    feature is enabled in the same environment.
+                    feature is{' '}
+                    <b>
+                        {parentValue.status === 'disabled'
+                            ? 'disabled'
+                            : 'enabled'}
+                    </b>{' '}
+                    in the same environment.
                 </Box>
 
-                <Typography>What feature do you want to depend on?</Typography>
+                <Typography>
+                    What <b>feature</b> do you want to depend on?
+                </Typography>
                 <ConditionallyRender
                     condition={showDependencyDialogue}
                     show={
                         <LazyOptions
                             project={project}
                             featureId={featureId}
                             parent={parent}
-                            onSelect={setParent}
+                            onSelect={(status) => {
+                                setParentValue({ status: 'enabled' });
```

Comment:
Maybe it should not reset the bottom configuration.

Reply:
I don't have a strong opinion here so let's gather feedback. I started w/o reset and it felt weird but it's just one opinion. For now it's hidden behind a flag anyway.